index.js

```diff
@@ -45,6 +46,31 @@ exports.parseAll = function(chtml, callback){
 };
 
 /**
+ * Exported function that takes html string and
+ * returns a BBPromise for all available metadata
+ *
+ * @param  {String}   html  	 html String HTML of the page
+ * @param  {Function} [callback] Optional callback
+ * @return {Object}              BBPromise for metadata
+ */
+exports.loadFromString = function(html, callback) {
+	return index.parseAll(cheerio.load(html)).nodeify(callback);
+};
+
+/**
+ * Exported function that takes html file and
+ * returns a BBPromise for all available metadata
+ *
+ * @param  {String}   path  	 path Path to HTML file
+ * @param  {Function} [callback] Optional callback
+ * @return {Object}              BBPromise for metadata
+ */
+exports.loadFromFile = function(path, callback) {
+	var html = fs.readFileSync(path, 'utf-8');
```

Comment:
Since we heavily rely on Promises, it would be better to have:

```js
return fs.readFileAsync(path, 'utf-8').then(html => index.parseAll(cheerio.load(html).nodeify(callback)));
```

Hm, thinking about it now, it would also be good to allow the caller to specify the file encoding, and use `utf-8` as the default if one is not provided.

Reply:
Oh, for this to work, the require needs to be `fs = BBPromise.promisify(require('fs'))`